### DISCUSSION TITLE
Archive scan detected malware!

### DISCUSSION THREAD

**Discussion by MrHinsh** [microsoft/winget-pkgs#106775] [General]
When I try to insatll my package locally to versify the manafest it stops at "Archive scan detected malware." and will not proceed unless I add `--ignore-local-archive-malware-scan`.

I am positive that I do not have any malware in this archive. How do I get more understanding of what it is detecting?

**Comment by Trenly** [comment]
I'm not sure if it answers your question, but the implementation specification has some information about [Zip Threat Detection](https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23140%20-%20ZIP%20Support.md#zip-threat-detection)

**Comment by soroshsabz** [comment]
ITNOA

How to find, why malware detect?
